po/ro.po

```diff
@@ -0,0 +1,703 @@
+msgstr "ID-ul de bază al pachetului"
+
+#: src/info.rs:76 src/info.rs:128
+msgid "Keywords"
+msgstr "Cuvinte cheie"
+
+#: src/info.rs:77 src/info.rs:130
+msgid "Snapshot URL"
+msgstr "Snapshot URL"
+
+#: src/info.rs:93
+msgid "No"
+msgstr "Nu"
+
+#: src/info.rs:113
+msgid "Optional Deps"
+msgstr "Dependințe Opționale"
+
+#: src/install.rs:131 src/install.rs:545 src/install.rs:944
+msgid "no architecture"
+msgstr "nicio arhitectură"
+
+#: src/install.rs:138
+msgid "can't build package as root"
+msgstr "nu se poate compila pachetul ca root"
+
+#: src/install.rs:142
+msgid "failed to parse srcinfo generated by makepkg"
+msgstr "nu s-a putut procesa srcinfo generat de makepkg"
+
+#: src/install.rs:171 src/install.rs:442
+msgid "Resolving dependencies..."
+msgstr "Se rezolva dependințele..."
+
+#: src/install.rs:220 src/install.rs:225
+msgid "failed to download sources"
+msgstr "nu s-au putut descărca sursele"
+
+#: src/install.rs:230 src/install.rs:246 src/install.rs:254
+msgid "failed to build"
+msgstr "nu s-a putut compila"
+
+#: src/install.rs:233
+msgid "parsing pkg list..."
+msgstr "se procesează lista pkg..."
+
+#: src/install.rs:261 src/install.rs:1462
+msgid "{}-{} is up to date -- skipping build"
+msgstr "{}-{} este actualizat la ultima versiune -- se sare peste compilare"
+
+#: src/install.rs:333 src/query.rs:50 src/lib.rs:84
+msgid "error:"
+msgstr "eroare:"
+
+#: src/install.rs:334
+msgid "could not get news"
+msgstr "nu s-au putut obține noutăți"
+
+#: src/install.rs:339 src/install.rs:521 src/install.rs:716 src/install.rs:848
+msgid "Proceed with installation?"
+msgstr "Continuă cu instalarea?"
+
+#: src/install.rs:356
+msgid "no targets specified (use -h for help)"
+msgstr "niciun target specificat (folosește -h pentru ajutor)"
+
+#: src/install.rs:434 src/install.rs:461 src/search.rs:324 src/search.rs:360
+#: src/lib.rs:227
+msgid " there is nothing to do"
+msgstr " nu este nimic de făcut"
+
+#: src/install.rs:488
+msgid "can't install AUR package as root"
+msgstr "nu se poate instala pachetul AUR ca root"
+
+#: src/install.rs:507
+msgid "Remove make dependencies after install?"
+msgstr "Elimina dependințele make după instalare?"
+
+#: src/install.rs:518
+msgid "Proceed to review?"
+msgstr "Continuă cu verificarea pachetului?"
+
+#: src/install.rs:559
+msgid "The following packages are not compatible with your architecture:"
+msgstr "Următoarele pachete nu sunt compatibile cu arhitectura ta:"
+
+#: src/install.rs:573
+msgid "Would you like to try build them anyway?"
+msgstr "Dorești să încerci să le compilezi oricum?"
+
+#: src/install.rs:662
+msgid "packages failed to build: {}"
+msgstr "pachetele nu au putut fi compilate: {}"
+
+#: src/install.rs:698
+msgid "could not find .SRINFO for '{}'"
+msgstr "nu s-a putut găsi .SRCINFO pentru '{}'"
+
+#: src/install.rs:762
+msgid "failed to read dir: {}"
+msgstr "nu s-a putut citi directorul: {}"
+
+#: src/install.rs:778
+msgid ""
+"{} is a directory\n"
+"\n"
+msgstr "{} este un director\n"
```

Comment:
Copy the formatting above.

Reply:
Fixed the formatting, thanks for pointing it out.